src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -346,20 +348,17 @@ bool AdaptDispatch::_CursorMovePosition(const std::optional<size_t> row, const s
         }
         else
         {
-            // remember, in VT speak, this is relative to the viewport. not absolute.
-            SHORT diff;
-            success = SUCCEEDED(ShortSub(csbiex.dwCursorPosition.Y, csbiex.srWindow.Top, &diff));
-            if (success)
-            {
-                success = SUCCEEDED(ShortToSizeT(diff, &rowActual));
-            }
+            cursor.Y = csbiex.dwCursorPosition.Y - csbiex.srWindow.Top; // remember, in VT speak, this is relative to the viewport. not absolute.
```

Comment:
and this one _doesn't_ need a safe subtract?

Reply:
OK fine. I'm trying to appease both you and @j4james at once. :P 

We'll save James's concern for a concerted effort LATER to not be dumb about this.